templates/vue/src/components/tyde/activities/CircleOfSupport/CommunityView/onboarding/Tooltip.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div
+    ref="tooltip"
+    class="tooltip-container"
+    :style="{ opacity: positioned ? 1 : 0 }"
+  >
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+// import Helpers from "@/utils/Helpers"
+export default {
+  props: {
+    /**
+     * ID of the DOM element that we want to position this tooltip against.
+     */
+    // refId: {
+    //   type: String,
+    //   required: false,
+    // },
+    position: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      positioned: false,
+    }
+  },
+  mounted() {
+    // const container = document.getElementById("cos")
+    // const target = container.querySelector(`#${this.refId}`)
+    const tooltip = this.$refs.tooltip
+
+    // Temporary fix of positioning
+    if (this.position === "right") {
+      tooltip.style.right = "22px"
+      tooltip.style.bottom = "95px"
+      tooltip.classList.add("tooltip-container-right")
+    } else if (this.position === "left") {
+      tooltip.style.left = "22px"
+      tooltip.style.bottom = "95px"
+      tooltip.classList.add("tooltip-container-left")
+    }
+
+    // const { x, y } = Helpers.positionTooltip(target, tooltip, container, {
+    //   dryRun: true,
+    // })
+    // tooltip.style.transform = `translate(${x}px, ${y -
+    //   tooltip.clientHeight -
+    //   target.clientHeight}px)`
```

Comment:
Not sure why this is commented out? If you want to leave commented out code, please also include a comment to explain why it's currently commented out.